src/Arcus.WebApi.Security/Authorization/Jwt/JwtTokenReader.cs

```diff
@@ -39,21 +43,33 @@ public JwtTokenReader(string applicationId)
         /// </summary>
         /// <remarks>Uses Microsoft OpenId connect discovery endpoint</remarks>
         /// <param name="tokenValidationParameters">Collection of parameters to influence how the token validation is done</param>
-        public JwtTokenReader(TokenValidationParameters tokenValidationParameters) : this(tokenValidationParameters, MicrosoftDiscoveryEndpoint)
+        /// <param name="claimCheck">Custom claims key-value pair to validate against</param>
+        public JwtTokenReader(TokenValidationParameters tokenValidationParameters, IDictionary<string, string> claimCheck = null) : this(tokenValidationParameters, MicrosoftDiscoveryEndpoint, claimCheck)
```

Comment:
We may consider a constructor overload instead of an optional parameter because it will result in `MethodNotFound` exceptions when the packages are updated sepeartly.

Reply:
done